src/main/java/uk/gov/hmcts/ccd/v2/internal/controller/UICaseSearchController.java

```diff
@@ -170,7 +168,7 @@ public UICaseSearchController(
             .withSearchRequest(searchRequest)
             .build();
 
-        CaseSearchResult caseSearchResult = caseSearchOperation.execute(request);
+        CaseSearchResult caseSearchResult = caseSearchOperation.execute(request, true);
```

Comment:
```suggestion
        CaseSearchResult caseSearchResult = caseSearchOperation.execute(request, false);
```
The internal search API doesn't currently return `data_classification` in the response so it can always be `false` in this scenario